course/enrollment.py

```diff
@@ -178,6 +179,15 @@ def decide_enrollment(approved, modeladmin, request, queryset):
             participation.status = participation_status.denied
         participation.save()
 
+        send_enrollment_decision(participation, approved, request)
+
+        count += 1
+
+    messages.add_message(request, messages.INFO,
+            # Translators: how many enroll requests have ben processed.
+            _("%d requests processed.") % count)
+
+def send_enrollment_decision(participation, approved, request):
         with translation.override(settings.RELATE_ADMIN_EMAIL_LOCALE):
```

Comment:
Yes. This message is cc'd to course staff though, and it's initiated from an HTTP request coming from a staff account, so the default locale seems like a fair fallback. The way I view it, both staff and students should be able to understand the default locale...